### PR TITLE
Add a Hand Drawn Tile

### PR DESCRIPTION
**Pull request by akshit15013**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Tried to [squash the commits][squash].
- [x] Added Screenshot to Indidate Location of Change in Game

### Changes done in this Pull Request

- View my work at https://pr-855-fossasia-labyrinth.surge.sh

### The problem I want to solve or the facility I want to improve is/are
Added a Hand Drawn Tile

### Screenshots, Location of Changes

screenshot 154

[squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

### CODE REVIEW

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE, CAN YOU PLEASE MERGE IT?

**Comment by akshit15013** [comment]
@YashKumarVerma

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE, COULD YOU PLEASE MERGE IT?

**Comment by YashKumarVerma** [comment]
the issue **_must follow the template_**

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE, COULD YOU PLEASE MERGE IT?

**Comment by akshit15013** [comment]
@YashKumarVerma

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE, COULD YOU PLAESE MERGE IT?

File: js/levels.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ function createFirstLevel()
 {
   return new Level("Starter", [
     [door.navypink, door.blood, door.marina, door.treasureKey, door.drawn, door.night],
-    [door.dark1, door.minecraftEntry, door.minecraft, door.orange, door.reddish, door.winter1, door.goal, door.none],
+    [door.dark1, door.minecraftEntry, door.minecraft, door.orange,door.drawnbyhand, door.reddish, door.winter1, door.goal, door.none],
```
Comment (abishekvashok) [inline js/levels.js]: Delete a tile which has occurred more than once and add yours.

**Comment by akshit15013** [comment]
@abishekvashok @YashKumarVerma

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE, COULD YOU PLEASE MERGE IT?

**Comment by akshit15013** [comment]
I have done these changes because Door.drawn was appearing more than once

**Comment by akshit15013** [comment]
@YashKumarVerma

**Pull request merged**